app/javascript/src/components/Dashboard/Notes/Item.jsx

```diff
@@ -39,13 +41,11 @@ const Item = ({ note, showDeleteAlertForNote, showEditNotePane }) => {
             />
           ))}
         </div>
-        {/* TODO: Add function to calculate times_ago for tooltip */}
-        <Tooltip content="Wednesday, 10:30 AM" position="bottom-start">
+        <Tooltip content={formatDate(note.createdAt)} position="bottom-start">
           <div className="flex items-center space-x-2">
             <Clock size="15" />
             <Typography className="flex items-start " style="body1">
-              {/* TODO: Add function to calculate times_ago */}
-              {note.status} 2 hours ago
+              {note.status} {timesAgo(note.createdAt)}
```

Comment:
I feel that the name `timesAgo` does not properly convey the required functionality. Don't you think something like `findElapsedTime` would be more suitable here?

Reply:
Yes, sounds much better.
Will do that.